### PR TITLE
#162240908 Users should be able to see their rating on an article

### PR DESCRIPTION
**Pull request by emmaadesile**
### What does this PR do?
Users should be able to see their rating on an article

### Description of tasks to be completed
- return a user's rating when the user rates an article

### How can this be manually tested?
`git clone <branchname>`
`npm install`
`npm run start_dev`
make a get request to `localhost:8000/api/v1/articles/<articleId>/user-rating`

### What are the relevant pivotal tracker stories
[162240908](https://www.pivotaltracker.com/story/show/162240908)

### CODE REVIEW

File: server/controllers/articleController.js [hunk 1]
```diff
@@ -237,6 +237,32 @@ const articlesController = {
       }
     });
   },
+
+  /**
+   * @description Get a user's ratint an an article
```
Comment (waleCloud) [inline server/controllers/articleController.js:242]: typo here

File: test/rateArticles.spec.js [hunk 2]
```diff
@@ -142,6 +142,19 @@ describe('ARTICLES RATING TESTS', () => {
...
+  });
+
+  it('gets a user\'s rating on an articles', (done) => {
+    chai
+      .request(app)
+      .get('/api/v1/articles/1/user-rating')
+      .set('authorization', token2)
+      .end((err, res) => {
+        expect(res.body.status).to.be.equal('success');
+        expect(res.body.data.userRating).to.equal(5);
```
Comment (waleCloud) [inline test/rateArticles.spec.js:157]: Why are you expecting the userRating to be 5?

File: server/controllers/articleController.js (hunk 1 above)
Comment (emmaadesile) [reply server/controllers/articleController.js:242]: I'll fix that now.

File: test/rateArticles.spec.js (hunk 2 above)
Comment (emmaadesile) [reply test/rateArticles.spec.js:157]: In the previous test-case, the article was rated 5 by the user. In this test-case, when I make a request to get the user's rating for that article, it should return 5.

File: server/controllers/articleController.js (hunk 1 above)
Comment (emmaadesile) [reply server/controllers/articleController.js:242]: Fixed

**Review by waleCloud — APPROVED** [review]

**Pull request merged**